vsintegration/src/vs/FsPkgs/FSharp.LanguageService/FSharp.LanguageService.Base/CodeWindowManager.cs

```diff
@@ -132,19 +124,15 @@ public class CodeWindowManager : IVsCodeWindowManager {
             }
         }
 #endif
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.DropDownHelper"]/*' />
         /// <summary>Return the optional TypeAndMemberDropdownBars object for the drop down combos</summary>
         internal TypeAndMemberDropdownBars DropDownHelper {
             get { return this.dropDownHelper; }
         }
-        /// <include file='doc\CodeWindowManager.uex' path='docs/doc[@for="CodeWindowManager.CodeWindow"]/*' />
         /// <summary>Return the IVsCodeWindow associated with this code window manager.</summary>
         internal IVsCodeWindow CodeWindow {
             get { return this.codeWindow; }
         }
       
-        #region IVsCodeWindowManager methods
```

Comment:
@latkin @KevinRansom i think we can use the explicit implementation, the methods become private/protected, but is better with all these interfaces